showcase/tests/integration/components/hds/accordion/index-test.gts

```diff
@@ -5,82 +5,99 @@
 
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'showcase/tests/helpers';
-import { click, render } from '@ember/test-helpers';
-import { hbs } from 'ember-cli-htmlbars';
+import { click, render, settled, find } from '@ember/test-helpers';
+import Accordion from '@hashicorp/design-system-components/components/hds/accordion/index';
```

Comment:
I do recognize this is done by codemod also probably up to HDS but it migth make sene to import this how it's being used on consumer side like

```suggestion
import { HdsAccordion, HdsItem } from '@hashicorp/design-system-components/components';
```

with that being said either way works fine